### PR TITLE
docs: Add `/healthz/readiness` endpoint

### PR DESCRIPTION
**Pull request by ivov**
Ref https://github.com/n8n-io/n8n/pull/10661

### CODE REVIEW

File: docs/hosting/logging-monitoring/monitoring.md [hunk 1]
```diff
@@ -9,16 +9,24 @@ contentType: howto
 There are two API endpoints you can call to check the status of your instance: `/healthz` and `/metrics`.
```
Comment (freakwriter) [inline docs/hosting/logging-monitoring/monitoring.md:9]:
```suggestion
There are three API endpoints you can call to check the status of your instance: `/healthz`, `healthz/readiness`, and `/metrics`.
```

File: docs/hosting/logging-monitoring/monitoring.md [hunk 2]
```diff
@@ -9,16 +9,24 @@ contentType: howto
...
 
 <!-- vale off -->
-## healthz
+## healthz and healthz/readiness
 <!-- vale on -->
-The `/healthz` endpoint returns a standard HTTP status code. 200 indicates the instance is reachable. It's available for both self-hosted and Cloud users.
+The `/healthz` endpoint returns a standard HTTP status code. 200 indicates the instance is reachable. It does not indicate DB status. It's available for both self-hosted and Cloud users.
```
Comment (freakwriter) [inline docs/hosting/logging-monitoring/monitoring.md:14]:
```suggestion
The `/healthz` endpoint returns a standard HTTP status code. 200 indicates the instance is reachable. It doesn't indicate DB status. It's available for both self-hosted and Cloud users.
```

**Review by freakwriter — CHANGES_REQUESTED** [review]
@ivov Thanks for this PR. Can you review the two suggested commits I added? The first is just to make sure the intro includes the new endpoint. The second is to make our linting tool happier by replacing "does not" with "doesn't".

**Comment by ivov** [comment]
LGTM 👍🏻

**Review by freakwriter — APPROVED** [review]

**Pull request merged**